web/less/landing.less

```diff
@@ -265,8 +267,7 @@ a {
       position: relative;
       font-size: 16px;
       width: 100%;
-      bottom: 98px;
-      border-bottom: solid 1px @flamingoRed;
+      bottom: 40px;
```

Comment:
I'm not sure what's causing it, but there is a lot of vertical space between the room name text field and the user name text field. I'd say 8px space would be better.